### ISSUE TITLE
[BITV] Accessibility issues

### ISSUE THREAD

**Issue by PVince81** [nextcloud/activity#898]
## Bug fixes
* [x] #1019
* [x] #1020
* [ ] Clicking on a file should ideally open the image or file in place: right now a redirection to files takes place. File will be opened "in place".

Following suggestions were moved to main "Lazy loading" https://github.com/nextcloud/server/issues/33844 issue: 
 - navigation via months/time could be useful
 - filtering/search would be useful

**Comment by nickvergessen** [comment]
> Better to use underline for hover/focus style of the links

Since activity is not reverting a server rule there, would it make sense to do this on a global level? Otherwise we also need to track this for all the links in settings descriptions, notifications, ...

**Comment by nickvergessen** [comment]
> Lazy loading not accessible as you can not move very far to the past quickly.

You can't anyway as the responses are rather slow and only get 30 entries per grab (could increase that now that the server load was improved with #871 

I split the issue list into "bug fixes" which we should do now and "follow up features" which can be done later. Let me know if you agree.

**Comment by PVince81** [comment]
yeah, lazy loading / moving to pagination is going to be a bigger challenge in multiple fronts anyway...

**Comment by PVince81** [comment]
yes, I've edited and moved it to features

**Comment by nickvergessen** [comment]
> Better to use underline for hover/focus style of the links

That should be applied to all Nextcloud? Or what makes it valid here, but e.g. not in notifications etc?

**Comment by st3iny** [comment]
Please check if some of the issues might be fixed by #1043.

**Comment by JuliaKirschenheuter** [comment]
> [ ] Clicking on a file should ideally open the image or file in place: right now a redirection to files takes place.

@michaelnissenbaum is this approach ok to you?

**Comment by michaelnissenbaum** [comment]
@JuliaKirschenheuter If by 'open in place' you mean that the image/file will be opened on the same page in a viewer/editor, then that's acceptable to me. It's important that the editor/viewer is fully accessible. Additionally, please ensure that once a file or image is opened, the keyboard focus is properly placed and clearly visible.
